.teamcity/src/subprojects/build/core/JavaScriptBuild.kt

```diff
@@ -23,10 +24,15 @@ class JavaScriptBuild(private val javaScriptEngine: JavaScriptEngine) : BuildTyp
             setupDockerForJavaScriptTests(javaScriptEngine)
         }
     }
+    features {
+        perfmon {
```

Comment:
Maybe better to have open and closed curly brackets on the same line if a block is empty: `perfmon {}`